packages/lib/server/queries/booking/index.ts

```diff
@@ -10,14 +10,15 @@ export const getTotalBookingDuration = async ({
   endDate: Date;
 }) => {
   // Aggregates the total booking time for a given event in a given time period
-  const [totalBookingTime] = (await prisma.$queryRaw`
+  // FIXME: bookings that overlap on one side will never be counted
+  const [totalBookingTime] = await prisma.$queryRaw<[{ totalMinutes: number | null }]>`
```

Comment:
previous type assertion missed this could be `null` which caused issues when comparing to numbers